WellTask_Frontend/src/components/base-component/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import classNames from "classnames";
+
+type ButtonType = "primary" | "secondary" | "outline";
+
+interface ButtonProps
+  extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, "type"> {
+  type?: ButtonType;
+  htmlType?: "submit" | "reset" | "button";
+  fullWidth?: boolean;
+  size?: "sm" | "md" | "lg";
+}
+
+export function Button({
+  children,
+  type = "primary",
+  htmlType,
+  fullWidth = false,
+  size = "md",
```

Comment:
```suggestion
  size = "medium",
```
Please use small, medium, large for sizing